bootstrap.d/x11-base.yml

```diff
@@ -152,3 +149,62 @@ packages:
       - args: ['ninja', 'install']
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
+
+  - name: xwayland
```

Comment:
Missing metadata, feel free to add it if you feel like maintaining this part of the Xorg stack. If you rather have me maintain it, let me know and I'll take it.